dags/probe_scraper.py

```diff
@@ -13,17 +15,37 @@
     'retry_delay': timedelta(minutes=30),
 }
 
-dag_daily = DAG('probe_scraper',
+with DAG('probe_scraper',
                 default_args=default_args,
-                schedule_interval='@daily')
+                schedule_interval='@daily') as dag:
+
+    probe_scraper = EMRSparkOperator(
```

Comment:
Since this is new, we may want to consider using dataproc

Reply:
This isn't new, I just moved it a bit. Let's keep it in AWS until it's ready for GCP.